lib/ruby/driver.rb

```diff
@@ -0,0 +1,111 @@
+require "ruby/driver/version"
+require 'active_support'
+require 'ripper'
+require 'yajl'
+
+module Ruby
+  module Driver
+      class MyException < StandardError
+      end
+
+      class RequestMessage
+          attr_reader :action
+          attr_reader :language
+          attr_reader :language_version
+          attr_reader :content
+          BAD_REQUEST = "Bad Request"
+          PARSE_AST = "ParseAST"
+
+          def initialize(json_req)
+                  check_req(json_req)
```

Comment:
if the `json_req` can be accessed with `:action` i'm assuming it's not a JSON as the variable suggests, but a hash. In that case, I'd change the variable name to something that does not imply it's a json, because it's not. What do you think?